<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">IQ聊天</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="themla">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px; 
background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string>账号:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:15px; 
background-color: rgb(226, 226, 226);</string>
    </property>
    <property name="text">
     <string>密码:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));
border-radius:50px; </string>
    </property>
    <property name="text">
     <string>IQ聊天</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="usele">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px; 
border:1px solid#000000;

</string>
    </property>
    <property name="placeholderText">
     <string>请输入账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px; 
border:1px solid#000000;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginG">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px; 
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registB">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px; 
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="auto_login">
       <property name="styleSheet">
        <string notr="true">border-radius:5px; 
background-color: rgb(226, 226, 226);</string>
       </property>
       <property name="text">
        <string>自动登录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="keep_passw">
       <property name="styleSheet">
        <string notr="true">border-radius:5px; 
background-color: rgb(226, 226, 226);</string>
       </property>
       <property name="text">
        <string>记住密码</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">border-radius:6px; 
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>忘记密码</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="softkeyboard">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/img/softkeyboard.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="SoftKeyBoard" name="skboard" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SoftKeyBoard</class>
   <extends>QWidget</extends>
   <header>softkeyboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
